python/ShapeTools.py

```diff
@@ -137,10 +141,11 @@ def doIndividualModels(self):
                     prop_b = customizer.build(True)
                     if len(sigcoeffs):
                         prop_b.SetName("prop_bin%s_bonly" % b)
-                    sum_b = ROOT.RooRealSumPdf("pdf_bin%s_bonly"       % b,  "", ROOT.RooArgList(prop_b),   ROOT.RooArgList(self.out.var('ONE')), True)
+                    self.objstore[prop_b.GetName()] = prop_b
```

Comment:
Is this bypassing the new `self.addObj` because it would be cumbersome /impossible to match the signature?